### PR TITLE
Ugdg update

### PR DESCRIPTION
**Pull request by kevinkuo0320**
Update DG an UG

### CODE REVIEW

File: docs/diagrams/ImportFeature.puml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+start
+:User inputs Import Command with file path;
+
+if () then ([File data follows correct format])
+    :Parse data into Json format;
+        :Returns Message to user indicating Import Success;
+        :ResultDisplay prints result to user;
+
+else ([else])
```
Comment (cwq2326) [inline docs/diagrams/ImportFeature.puml:12]:
```suggestion
else ([Else])
```
The first character can be capitalise for consistency

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -998,6 +1014,37 @@ When a user inputs `resize 1`, `MainWindow#executeCommand()` will be invoked whi
 This will then create a `ResizeCommand` object which is then executed by the `LogicManager` via `ResizeCommand#execute()` which will then update the value needed to set the result display window size. This value is used by the `MainWindow#handleResizeResultDisplayWindow()`
 which sets the window in the GUI according to the user's desired option.
 
+--------------------------------------------------------------------------------------------------------------------
+
+###Potential Feature: Import
```
Comment (cwq2326) [inline docs/DeveloperGuide.md:1019]:
```suggestion
### Potential Feature: Import 
```

Missing a spacing here

**Review by cwq2326 — CHANGES_REQUESTED** [review]

**Review by cwq2326 — APPROVED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -597,6 +637,29 @@ there are changes to Tracey.
 
 --------------------------------------------------------------------------------------------------------------------
```
Comment (tanchinkiat99) [inline docs/UserGuide.md:639]: I think this appendix thingy not supposed to be here

File: docs/UserGuide.md [hunk 4]
```diff
@@ -422,6 +422,46 @@ From *Figure 12*, this function is beneficial for administrators as you may want
...
+The `EditCommand` object is then passed back to the `LogicManager` which will then invoked `EditCommand#execute()`. This execute method will call two other helper methods `EditCommand#editChecker()` and `Person#isDifferentPerson()`, which both are not shown in the sequence diagram and is used further validation.
+The main functions of these two methods are to check if the new values are duplicate of the corresponding fields to be edited and if the new values for fields that requires uniqueness already exists in the address book respectively.
+The `ObservableList` in the `Model` class then updates the display of the contacts, placing the edited person to the bottom of the list (or placing it at the last index).
+
+TThe `ObservableList` is a JavaFX class which observes and automatically changes the list once an update is performed.
+
+### Filter feature
```
Comment (tanchinkiat99) [inline docs/UserGuide.md:434]: Filter feature shouldnt be here!

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -491,39 +493,53 @@ Afterwards, the `execute()` method of this ClearCommand object is called, which
...
+The find feature allows users to find a particular contact by its name in Tracey and retrieve their specific details.
 
-#### Usage Scenario
+The `find` command is as follows:
 
-Given below is an example usage scenario and how `find` react and act at each step.
+`find NAME`
 
-**1**) The user launches the application for the first time.
+The user do not require to search the full name of the contact. However,  the query substring must match the person's
```
Comment (DannyDakota) [inline docs/DeveloperGuide.md:506]: is not required*

File: docs/UserGuide.md [hunk 6]
```diff
@@ -597,6 +637,29 @@ there are changes to Tracey.
 
 --------------------------------------------------------------------------------------------------------------------
 
+## **Appendix: Instructions for manual testing**
```
Comment (DannyDakota) [inline docs/UserGuide.md:640]: This is in DG

**Pull request merged**